### PR TITLE
Change wording for resource interruption

### PR DESCRIPTION
**Pull request by fpagnoux**
Proposition : 
 
image

(La checkbox est décochée par default)

### CODE REVIEW

File: app/js/directives/captureMontantRessource.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 'use strict';
 
 angular.module('ddsApp').directive('montantRessource', function(SituationService) {
+    function getFormattedLabel (ressourceLabel) {
+        return (/^[aeiouy]/i.test(ressourceLabel) ? 'd’' : 'de ') + ressourceLabel.slice(0,1).toLowerCase() + ressourceLabel.slice(1)
```
Comment (MattiSG) [inline app/js/directives/captureMontantRessource.js]: Semicolon.

Comment (MattiSG) [inline app/js/directives/captureMontantRessource.js]: Good job. It's missing accent folding though, might be worth adding it now..

**Comment by MattiSG** [comment]
GTM

File: app/js/directives/captureMontantRessource.js (hunk 1 above)
Comment (fpagnoux) [inline app/js/directives/captureMontantRessource.js]: Ok, is it enough ?

**Comment by MattiSG** [comment]
GTM

**Pull request merged**